woodwork/tests/accessor/test_statistics.py

```diff
@@ -506,6 +506,129 @@ def test_dependence_dropna():
     assert mi_df.equals(expected_df)
 
 
+@pytest.mark.parametrize(
+    "logical_types",
+    [{"a_column": "Unknown"}, {"a_column": "Categorical"}],
+)
+def test_dependence_drop_columns(logical_types):
```

Comment:
Great test coverage! Since you're using the tag `category` here `categoricals = datatable.ww.select("category").columns`, would it be helpful to also include a Logical Type that has a primary tag of `category`, like `Ordinal` or `SubRegionCode`?